File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/SubScriptionModel.java
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels;

public class SubScriptionModel {
    boolean active_pay_button;
    boolean is_active_sub;
    boolean is_paid_sub;
    String user_allowed;
    String message;
    String month;
    String email;

//    public SubScriptionModel(boolean active_pay_button, boolean is_active_sub, boolean is_paid_sub, String email, String user_allowed, String message, String month) {
//        this.active_pay_button = active_pay_button;
//        this.is_active_sub = is_active_sub;
//        this.is_paid_sub = is_paid_sub;
//    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getMonth() {
        return month;
    }

    public void setMonth(String month) {
        this.month = month;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getUser_allowed() {
        return user_allowed;
    }

    public void setUser_allowed(String user_allowed) {
        this.user_allowed = user_allowed;
    }

    public boolean isActive_pay_button() {
        return active_pay_button;
    }

    public void setActive_pay_button(boolean active_pay_button) {
        this.active_pay_button = active_pay_button;
    }

    public boolean isIs_active_sub() {
        return is_active_sub;
    }

    public void setIs_active_sub(boolean is_active_sub) {
        this.is_active_sub = is_active_sub;
    }

    public boolean isIs_paid_sub() {
        return is_paid_sub;
    }

    public void setIs_paid_sub(boolean is_paid_sub) {
        this.is_paid_sub = is_paid_sub;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/SubScriptionModel.kt
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels

class SubScriptionModel {
    var active_pay_button: Boolean = false
        @JvmName("isActive_pay_button") get
        @JvmName("setActive_pay_button") set

    var is_active_sub: Boolean = false
        @JvmName("isIs_active_sub") get
        @JvmName("setIs_active_sub") set

    var is_paid_sub: Boolean = false
        @JvmName("isIs_paid_sub") get
        @JvmName("setIs_paid_sub") set

    var user_allowed: String? = null
    var message: String? = null
    var month: String? = null
    var email: String? = null
}
